models/repo/repo.go

```diff
@@ -513,7 +513,6 @@ func (repo *Repository) DescriptionHTML(ctx context.Context) template.HTML {
 	desc, err := markup.RenderDescriptionHTML(&markup.RenderContext{
 		Ctx:       ctx,
 		URLPrefix: repo.HTMLURL(),
```

Comment:
```suggestion
		URLPrefix: repo.HTMLURL(),
		// Don't use Metas to speedup requests
```

we might want to add a note ...